includes/country-functions.php

```diff
@@ -1658,3 +1658,18 @@ function edd_get_spain_states_list() {
 	return apply_filters( 'edd_spain_states', $states );
 }
 
+/**
+ * Given a country code, return the country name
+ *
+ * @since 2.8.7
+ * @param string $country_code The ISO Code for the country
+ *
+ * @return string
+ */
+function edd_get_country_name( $country_code = '' ) {
+	$country_list = edd_get_country_list();
+	$country_name = isset( $country_list[ $country_code ] ) ? $country_list[ $country_code ] : $country_code;
+
+	return apply_filters( 'edd_country_label', $country_name, $country_code );
```

Comment:
Can we please make the filter name match the function name?

Reply:
Gah sorry It did at one point, just decided to change the function name :)